sdmetrics/reports/single_table/diagnostic_report.py

```diff
@@ -178,6 +181,22 @@ def get_visualization(self, property_name):
             plotly.graph_objects._figure.Figure
                 The visualization for the requested property.
         """
+        score_breakdowns = {
+            metric.__name__: self._metric_results[metric.__name__]
+            for metric in self.METRICS.get(property_name, [])
+        }
+
+        if property_name == 'Coverage':
+            fig = get_column_coverage_plot(score_breakdowns, self._property_scores[property_name])
+
+        elif property_name == 'Boundaries':
+            fig = get_column_boundaries_plot(
+                score_breakdowns, self._property_scores[property_name])
+
+        elif property_name == 'Synthesis':
+            fig = get_synthesis_plot(score_breakdowns.get('NewRowSynthesis', {}))
+
+        return fig
```

Comment:
I think it would be nice to have a warning if `fig` is `None`. (I think we should define `fig = None` at the top, then if `fig` is `None` let the user know the allowed properties. If a `property` that is not one of the three above, we produce an `UnboundLocalError`.